### ISSUE TITLE
Add showDelayBeforeDeletionPopup (bool) as parameter

### ISSUE THREAD

**Issue by novoid** [ether/etherpad#5670]
* * *

name: Feature request
about: Suggest an idea for this project
title: ''
labels: enhancement
assignees: 

* * *

**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I'm using EtherPad on a mobile phone browser. By default, the popup is shown on each and every invocation of a Pad page. On a mobile browser, it takes up most of the content as long as the keyboard is visible as well.

https://etherpad.org/doc/v1.8.18/#index_embed-parameters isn't listing any option here.

**Describe the solution you'd like**
Whatever popup is configured, a user should have the possibility to disable them as page parameter in order to customize the behavior for minimizing frustration.

**Describe alternatives you've considered**
-

**Additional context**
-

**Plugin?**
-

**Comment by JohnMcLear** [comment]
Just a modification to swagger.  Did you figure this out?

**Comment by JohnMcLear** [comment]
It'd be nice to have an option to "Mute chat" or "Focused mode" where Swagger doesn't pop up..  How about a "Focused mode" that is a user setting that disables all swagger popups?  Sounds nice to me :)  That or if you open chat a little speaker where you can click it to mute chat..  

I assume by popup you just mean chats?  I don't really see swagger used that much other than chat..  IDK if you click that if its persistent per pad or per user or not..  Hard to say exactly :)

**Comment by JohnMcLear** [comment]
Revisiting this four years on — the generic ask ("let me disable page-load popups via URL parameter") still makes sense, but the specific `showDelayBeforeDeletionPopup` name doesn't match anything on `develop` today: there's no pad-deletion popup on page load in the current code.

There *is* a pad-deletion-token modal being built on the `feat-gdpr-pad-deletion` branch (shown once to the creator on first visit via `src/static/js/pad.ts` `showDeletionTokenModalIfPresent`). Once that lands, it would be reasonable to add a `showDeletionToken=false` URL param that skips it for integrators who want a quieter embed.

Holding off until that branch merges so the param plumbing ends up in one place. Will circle back then.
